client/landing/gutenboarding/onboarding-block/design-selector/design-card.tsx

```diff
@@ -20,13 +20,22 @@ interface Props {
 	design: import('@automattic/data-stores').VerticalsTemplates.Template;
 	onClick: MouseEventHandler< HTMLDivElement >;
 	style?: CSSProperties;
+	dialogId: string;
 }
-const DesignCard: FunctionComponent< Props > = ( { design, onClick, isSelected, style } ) => (
+const DesignCard: FunctionComponent< Props > = ( {
+	design,
+	dialogId,
+	isSelected,
+	onClick,
+	style,
+} ) => (
 	<Card
 		className={ classnames( 'design-selector__design-option', { 'is-selected': isSelected } ) }
 		isElevated
 		onClick={ onClick }
 		style={ style }
+		aria-haspopup="dialog"
+		aria-controls={ dialogId }
```

Comment:
There's no value in adding those aria attributes if the component is not accessible.

> Use Reakit Rover for tab access to designs and templates in the page selector.

I'd not use `Rover` here. `Rover` will apply the roving tabindex method to the cards, which means that, by default, users won't be able to <kbd>Tab</kbd> through the cards. And since they have no clue that they should use arrow keys (as they would with widgets like toolbar, menu etc.), keyboard users may still not be able to navigate through this page.

That could be solved by [overwriting the `tabIndex` prop rendered by `Rover`](https://codesandbox.io/s/reakit-tabbable-rover-ciw69), which would allow users to <kbd>Tab</kbd> and use arrow keys at the same time! 🙌

But that's not the only problem! 😔

`Rover` only adds roving tabindex in one dimension. Sighted users using the keyboard could expect that they could navigate the card list as if it were a grid (bidimensional), since that's what they see. There's an ongoing conversation around adding a `Grid` module to Reakit: https://github.com/reakit/reakit/issues/435

Instead of overcomplicating this, I'd just make this component tabbable.

[`Tabbable`](https://reakit.io/docs/tabbable/), which is used by `Rover` underneath, may help with this, specially because this component is a `div`. `Tabbable` would also make it clickable with <kbd>Enter</kbd> and <kbd>Space</kbd>.

A simpler (and better?) solution though is to just turn this element into a `button`. `<Card as="button"` should do the trick I guess.